### ISSUE TITLE
[CoE Starter Kit - Feature]: Role assignments for Apps & Flows

### ISSUE THREAD

**Issue by bhavikj65** [microsoft/coe-starter-kit#6307]
### Is your feature request related to a problem? Please describe.

We are working to implement some governance processes for Power Apps and for that it is important to know the role assignments for each app. Just with the Users Shared, Groups Shared and Total Shared it doesn't help us in taking a decision for orphaned apps. 

If we have role assignments it makes it easier to take a decision on orphaned apps, we can then identify users who are co-owners and then promote one of them as primary owner. 

It would be nice to have similar information for cloud flows so that we could act on cloud flows which are orphaned. 

### Describe the solution you'd like

Basically there should be a table which will store all role assignments for Apps & Flows (even separate table should work).

For example, we get all role assignments of an app and then for each value returned, we store that information in a table. 

### Describe alternatives you've considered

Currently, we have built a custom cloud flow that gets all role assignments for all apps and then store it in a custom table. However this is not very reliant because we have to build another flow which will mark apps as deleted when they are deleted  or any other changes that happens to the app.

**Comment by Jenefer-Monroe** [comment]
Hello. We do have this on our backlog https://github.com/microsoft/coe-starter-kit/issues/1083
I think we might be unblocked now to pursue but not sure. Will follow up to let you know before I close here.

**Comment by bhavikj65** [comment]
This item is more about flows, do you also have this for Power App's in
your backlog?

**Comment by Jenefer-Monroe** [comment]
oh I see. Power Apps we already have information in this table:
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/d6f11499-1e99-4b02-b77e-c0fb6e5a5d85)

**Comment by bhavikj65** [comment]
Great, it would be nice to have it for cloud flows as well.
